packages/keymaps/src/browser/keymaps-frontend-contribution.ts

```diff
@@ -63,6 +67,7 @@ export class KeymapsFrontendContribution extends AbstractViewContribution<Keybin
         });
         commands.registerCommand(KeymapsCommands.OPEN_KEYMAPS_JSON, {
             isEnabled: () => true,
+            isVisible: widget => this.isKeybindingWidget(widget),
```

Comment:
There is a bug where we cannot run the command `Settings: Open Keyboard Shortcuts (JSON)` from the command palette unless the keybindings-widget is currently opened.

**Reproduce:**
1. attempt to open the JSON file using <kbd>F1</kbd> → `Settings: Open Keyboard Shortcuts (JSON)`
2. notice that the command is **not present** in the quick-commands menu
3. open the keybindings-widget
4. repeat step 1
5. notice that the command is now **present** in the quick-commands menu

Reply:
Thanks Vince, I will look into it rn

Reply:
Just fixed this by separating command palettte & toolbar commands.